.gitlab-ci.yml

```diff
@@ -275,7 +275,7 @@ freebsd_12_release:
     MAKE: 'gmake'
   script:
     - ${GITLAB_MAKE} vms_start
-    - ${GITLAB_MAKE} vms_test_freebsd
+    - ${GITLAB_MAKE} vms_test_freebsd_no_deps
```

Comment:
I would leave some comment, which states that given image has foo, bar and baz installed.